### PR TITLE
Document how to look at logs with GCP

### PR DESCRIPTION
**Pull request by yuvipanda**
unlike k8s logs, these provide logs for upto the last 30 days.

Ref https://github.com/2i2c-org/infrastructure/issues/1890

### CODE REVIEW

**Review by sgibson91 — APPROVED** [review]
We should add a cross-ref to the ["Common problems and their solutions" page](https://infrastructure.2i2c.org/en/latest/sre-guide/common-problems-solutions.html) that says something like "For more general tips, see our Troubleshooting guide". Other than that, this is good to go!

File: docs/howto/troubleshoot/logs.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+```{tip}
+Another super-quick shortcut is to replace any `-` in the username with `-2d`,
+any `.` with `-2e` and any `@` with `-40`. If your username contains more
+special characters, highly recommend using the script instead - escaping
+errors can be frustrating!
+```
+
+#### Look at dask-gateway logs
+
+The following query will show logs from all the components of dask-gateay
```
Comment (consideRatio) [inline docs/howto/troubleshoot/logs.md:91]:
```suggestion
The following query will show logs from all the components of dask-gateway
```

**Review by consideRatio — APPROVED** [review]
Super helpful to have this level of detail on the examples!

I would have hesitated try to get logs from google's web UI before reading this, but not any more when this resources is available!

**Comment by jbusecke** [comment]
This seems super helpful for somebody like me. Thanks for putting in the effort.

**Pull request merged**